### DISCUSSION TITLE
How to use overwrite_element_solution_record functionality in ansys.mapdl.reader?

### DISCUSSION THREAD

**Discussion by fceyhuns** [pyansys/pymapdl-reader#267] [Q&A] [answered]
Hello, 

ansys.mapdl.reader is obviously to read Ansys results. Is there any package to overwrite results? 
I found overwrite_element_solution_record function which updates values in a session but I could not save the updated values. 
What is the proper usage of this function?

**Comment by mikerife** [comment, marked as answer]
Hi @fceyhuns there is no function in PyMAPDL Reader for this (it's a reader, not a writer). Instead check out the 'reswrite' of PyMAPDL (or in the MAPDL Help).  This can write a new MAPDL result file from what is stored in memory.  Or overwrite an existing result file.  Please be very careful with this function and be sure to document, document, document!  Mike
